ports/atmel-samd/boards/adafruit_proxlight_trinkey_m0/mpconfigboard.mk

```diff
@@ -0,0 +1,33 @@
+USB_VID = 0x239A
+USB_PID = 0x8104
+USB_PRODUCT = "ProxLight Trinkey M0"
+USB_MANUFACTURER = "Adafruit Industries LLC"
+
+CHIP_VARIANT = SAMD21E18A
+CHIP_FAMILY = samd21
+
+INTERNAL_FLASH_FILESYSTEM = 1
+LONGINT_IMPL = NONE
+
+CIRCUITPY_ANALOGIO = 0
+CIRCUITPY_ROTARYIO = 0
+CIRCUITPY_RTC = 0
+CIRCUITPY_SAMD = 0
+CIRCUITPY_PS2IO = 0
+CIRCUITPY_PULSEIO = 0
+CIRCUITPY_PWMIO = 0
+CIRCUITPY_AUDIOCORE = 0
+CIRCUITPY_BUSIO = 1
+CIRCUITPY_STORAGE = 0
```

Comment:
If you turn off `storage`, people can't do `storage.erase_filesystem()`, or remount to store stuff locally.